README.adoc

```diff
@@ -2537,18 +2544,29 @@ class Person
 end
 ----
 
-==== Non-declarative Methods That Have "keyword" Status in Ruby [[method-invocation-parens-non-declarative-keyword]]
+===== Non-Declarative Methods That Have "keyword" Status in Ruby [[method-invocation-parens-non-declarative-keyword]]
+
+There are two schools of thought within the community: one says to always omit parentheses for non-declarative methods that have "keyword" status (e.g., `Kernel` instance methods), and the other says not to omit them.
```

Comment:
The are two options indeed, but one is much more popular, so I guess we'd still have to recommend it and just mention the other.

Reply:
OK, I force-pushed an amended commit that makes it clear that one is much more popular.